services/editor/src/components/common/ComboBox/AutoSuggest.js

```diff
@@ -8,10 +8,11 @@ const mapSuggestionsToProps = mapPropsStream((props$) => {
 
   const query$ = Rx.Observable.merge(onSearch$, props$.pluck('value'));
 
-  const suggestions$ = query$.withLatestFrom(props$, (value, { getSuggestions }) => ({ value, getSuggestions }))
-    .distinctUntilKeyChanged('value')
-    .debounce(({ value }) => Rx.Observable.empty().delay(value === '' ? 0 : 500))
-    .switchMap(({ getSuggestions, value }) =>
+  const suggestions$ = query$
+    .withLatestFrom(props$.pluck('getSuggestions'), Array.of)
```

Comment:
the Array.of  trick with distinctUntilKeyChanged(0) is far less clear then the object version